### DISCUSSION TITLE
Native wind doesn't work properly

### DISCUSSION THREAD

**Discussion by elalami-m** [obytes/react-native-template-obytes#360] [Q&A] [answered]
I'm experiencing an issue with NativeWind where the w-40 class, which should apply a width of 160px, doesn't provide the same result as when I set an inline style with the same width value.

**Comment by yjose** [comment, marked as answer]
@elalami-m It seems there are some issues with the new NativeWind version. I recommend migrating to the [latest version](https://github.com/nativewind/nativewind/commits/main/). Currently, we are just waiting for the stable version to migrate to it.
If you migrate to the latest version and your issue still persists, please open an issue on the NativeWind repository.
